spec/lib/build_strategies/production_build_strategy_spec.rb

```diff
@@ -115,7 +115,6 @@
     }
 
     before do
-      allow(GitRepo).to receive(:inside_copy).and_yield
```

Comment:
`inside_repo` is already stubbed in the setup for the file. It's not obvious I know.